src/appleseed/foundation/meta/tests/test_microfacet.cpp

```diff
@@ -451,7 +451,7 @@ TEST_SUITE(Foundation_Math_Microfacet)
             for (size_t j = 0; j < num_sample; ++j)
             {
                 static const size_t Bases[] = { 2 };
-                const Vector2f s = hammersley_sequence<float, 2>(Bases, num_sample, i);
+                const Vector2f s = hammersley_sequence<float, 2>(Bases, num_sample, (size_t)i);
```

Comment:
Confirmed to be a bug, @gospodnetic is currently fixing it. Can you revert these changes?